lib/WebpackOptionsDefaulter.js

```diff
@@ -206,7 +207,13 @@ class WebpackOptionsDefaulter extends OptionsDefaulter {
 			options => options.experiments.outputModule
 		);
 		this.set("output.iife", "make", options => !options.output.module);
-		this.set("output.ecmaVersion", 2015);
+		this.set("output.ecmaVersion", "call", value => {
+			if (value) {
+				return normalizeEcmaVersion(value);
```

Comment:
Would be cool to seen an test in `test/configCases` which passes an `ecmaVersion`